layouts/partials/footer_js.html

```diff
@@ -1,3 +1,18 @@
+<script>
+$( ".rating label" ).click(function() {
+    const fieldset = document.querySelector('#page-rating fieldset');
+    const pageURL = document.location.pathname + document.location.search;
+    const rating =  $("#" + $(this).attr('for')).val();
+
+    if (!fieldset.classList.contains('disabled')) {
```

Comment:
Just in case they're using a browser extension that (poorly) blocks Google Analytics I would change this to:

```suggestion
    if (!fieldset.classList.contains('disabled') && typeof ga === "function") {
```